### PR TITLE
cloud: billing cost explorer

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

TiDB Cloud supports Billing > Cost Explorer.
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/tidb-cloud-billing.md [hunk 1]
```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
 > - The total amount in the monthly bill is rounded off to the 2nd decimal place.
 > - The total amount in the daily usage details is accurate to the 6th decimal place.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:82]: https://docs.pingcap.com/tidbcloud/manage-user-access#organization-role

Do we also need to add the info about viewing usage costs here?

image

File: tidb-cloud/tidb-cloud-billing.md [hunk 2]
```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
...
+
+To analyze and customize your cost reports of your organization, perform the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
+
+    > **Note:**
+    >
+    > If you are in multiple organizations, switch to your target organization by clicking its name.
+
+2. In the left navigation pane, navigate to **Billing > Cost Explorer**.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:95]: 这里是不是要先在左下角点 Billing
image

File: tidb-cloud/tidb-cloud-billing.md [hunk 3]
```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
...
+3. On the **Cost Explorer** page, expand the **Filter** section in the upper-right corner to customize your report. You can set the time range, select a grouping option (such as by service, project, cluster, region, product type, and charge type), and apply filters by selecting specific services, projects, clusters, or regions. The cost explorer will display you with the following information:
+
+    - **Cost Graph**: visualizes the cost trends over the selected time range. You can switch between **Monthly**, **Daily**, and **Total** views.
+    - **Cost Breakdown**: displays a detailed breakdown of your costs according to the chosen grouping option. For further analysis, you can download the data in CSV format.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:99]:
```suggestion
    - **Cost Breakdown**: displays a detailed breakdown of your costs according to the selected grouping option. For further analysis, you can download the data in CSV format.
```

File: tidb-cloud/tidb-cloud-billing.md (hunk 2 above)
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-billing.md:95]:
```suggestion
2. In the left navigation pane, click the **Billing** tab and then click the **Cost Explorer** tab.
```

File: tidb-cloud/tidb-cloud-billing.md (hunk 1 above)
Comment (Oreoxmt) [reply tidb-cloud/tidb-cloud-billing.md:82]: https://github.com/pingcap/docs/pull/16608/commits/2f0b028936c729833deada833e7e874bc464d962

File: tidb-cloud/tidb-cloud-billing.md [hunk 4]
```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
 > - The total amount in the monthly bill is rounded off to the 2nd decimal place.
 > - The total amount in the daily usage details is accurate to the 6th decimal place.
 
+## Cost explorer
+
+If you are in the `Organization Owner` or `Organization Billing Admin` role of your organization, you can view and analyze the usage costs of TiDB Cloud. Otherwise, skip this section.
+
+To analyze and customize your cost reports of your organization, perform the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of the TiDB Cloud console.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:89]:
```suggestion
1. In the lower-left corner of the [TiDB Cloud console](https://tidbcloud.com), click <MDSvgIcon name="icon-top-organization" />, and then click **Billing**.
```

File: tidb-cloud/tidb-cloud-billing.md [hunk 5]
```diff
@@ -80,6 +80,24 @@ The billing details page shows the billing summary by project and by service. Yo
...
+
+    > **Note:**
+    >
+    > If you are in multiple organizations, switch to your target organization by clicking its name.
+
+2. In the left navigation pane, click the **Billing** tab and then click the **Cost Explorer** tab.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-billing.md:95]:
```suggestion
2. On the **Billing** page, click the **Cost Explorer** tab.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**